starrocks/install.sh

```diff
@@ -5,3 +5,8 @@ sudo apt-get update
 sudo apt-get install -y mysql-client
 
 docker run -p 9030:9030 -p 8030:8030 -p 8040:8040 -itd --name starrocks starrocks/allin1-ubuntu
+
+
+# Enable FlatJSON 
+mysql -h 127.0.0.1 -P9030 -u root -e "SET GLOBAL cbo_prune_json_subfield = true";
```

Comment:
So here, we are applying non-default settings, right? To compare all systems on a fair basis, the benchmark is supposed to run with default settings.

Reply:
Yes, that would make more sense, but it is really up to you of course.